Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md

```diff
@@ -177,62 +180,69 @@ spec:
 
 ### Metadata
 
-* `name`: The name of the pool to create.
-* `namespace`: The namespace of the Rook cluster where the pool is created.
+-   `name`: The name of the pool to create.
+-   `namespace`: The namespace of the Rook cluster where the pool is created.
 
 ### Spec
 
-* `replicated`: Settings for a replicated pool. If specified, `erasureCoded` settings must not be specified.
-    * `size`: The desired number of copies to make of the data in the pool.
-    * `requireSafeReplicaSize`: set to false if you want to create a pool with size 1, setting pool size 1 could lead to data loss without recovery. Make sure you are *ABSOLUTELY CERTAIN* that is what you want.
-    * `replicasPerFailureDomain`: Sets up the number of replicas to place in a given failure domain. For instance, if the failure domain is a datacenter (cluster is
-stretched) then you will have 2 replicas per datacenter where each replica ends up on a different host. This gives you a total of 4 replicas and for this, the `size` must be set to 4. The default is 1.
-    * `subFailureDomain`: Name of the CRUSH bucket representing a sub-failure domain. In a stretched configuration this option represent the "last" bucket where replicas will end up being written. Imagine the cluster is stretched across two datacenters, you can then have 2 copies per datacenter and each copy on a different CRUSH bucket. The default is "host".
-* `erasureCoded`: Settings for an erasure-coded pool. If specified, `replicated` settings must not be specified. See below for more details on [erasure coding](#erasure-coding).
-    * `dataChunks`: Number of chunks to divide the original object into
-    * `codingChunks`: Number of coding chunks to generate
-* `failureDomain`: The failure domain across which the data will be spread. This can be set to a value of either `osd` or `host`, with `host` being the default setting. A failure domain can also be set to a different type (e.g. `rack`), if the OSDs are created on nodes with the supported [topology labels](../Cluster/ceph-cluster-crd.md#osd-topology). If the `failureDomain` is changed on the pool, the operator will create a new CRUSH rule and update the pool.
+-   `replicated`: Settings for a replicated pool. If specified, `erasureCoded` settings must not be specified.
+    _ `size`: The desired number of copies to make of the data in the pool.
```

Comment:
Are undescores accepted, or they should be all dashes?

Reply:
I think it must have thought that the `*` intended to italicize the line or something. I fixed it to be a dash, which is what Prettier uses.